src/components/Routes.js

```diff
@@ -25,7 +25,7 @@ const Routes = () => (
       }}
     />
     <Route exact path="/" component={LandingPage} />
-    <Route exact path="/" component={Welcome} />
+    <Route exact path="/welcome" component={Welcome} />
```

Comment:
Just realized it, but this welcome page was just a placeholder awaiting the landing page, now we can do away with it altogether. What do you think?